src/resolver.ts

```diff
@@ -20,8 +20,7 @@ export const createResolver = (resFn, errFn) => {
       });
     });
   };
-
-  baseResolver.createResolver = (cResFn, cErrFn) => {
+  baseResolver['createResolver'] = (cResFn, cErrFn) => {
```

Comment:
What is the reason we are defining an object property this way?

Reply:
In typescript we cannot directly add property to function, either we have to do it via namespace, or use conventional method of adding property, which I used above